exp/anbox-cloud.md

```diff
@@ -31,62 +31,82 @@ See the following table for a comparison of features for the different variants:
-
 [note type="information" status="Tip"]
-We recommend to always start with the Anbox Cloud Appliance for first tests. You can then expand to a full Anbox Cloud installation later.
+We recommend to always start with the Anbox Cloud Appliance. You can expand to a full Anbox Cloud installation later.
 [/note]
 
-## Components
+## Anbox Cloud Components
 
-Anbox Cloud is composed of different components that interact with one another.
+Anbox Cloud consists of the core stack, the streaming stack, and user-specific components that complete the streaming stack. The core stack includes all components that are required for a basic deployment of Anbox Cloud and also some additional, optional components. The streaming stack includes components which are optional but required for streaming.
 
-The core stack includes all components that are required for a basic deployment of Anbox Cloud to work. The streaming stack is a set of components needed to provide streaming functionality.
-For containerisation, Anbox Cloud includes LXD.
+An Anbox Cloud cluster can consist of the core stack or if streaming functionality is required, the streaming stack along with the core stack. An Anbox Cloud cluster can have multiple subclusters. The components inside a subcluster are explained in detail further in this topic.
 
-### Core stack
+The diagrams used to explain the architecture in this topic show the recommended way of collocating components. However, it is possible to collocate more components on the same machine, or to have separate machines for components that are collocated in the picture.
 
-Anbox Cloud takes care of all management aspects and provides both a fully functional implementation and an integration model to support existing infrastructure and service implementations.
+### Core stack
 
-The following figure gives an overview of the different components and their responsibility within the core stack of Anbox Cloud.
+The diagram below depicts the core stack and its components:
 
 ![Anbox Cloud core stack|690x398](https://assets.ubuntu.com/v1/e74d1a49-anbox_cloud_core-stack.png)
 
-At the heart of Anbox Cloud sits the [**Anbox Management Service (AMS)**](https://discourse.ubuntu.com/t/about-ams/24321). **AMS** has the job to handle all aspects of the application and container life cycle (including application and image updates) while ensuring high density, performance and fast container startup times.
+The core stack contains an Anbox subcluster, a Juju controller, and the Anbox Application Registry (AAR).
 
-A developer or system administrator will manage **AMS** through the **command line interface (AMC)** or through custom-built tools interacting with the [**AMS HTTP API**](https://discourse.ubuntu.com/t/ams-rest-api-reference/17801).
+Inside the Anbox subcluster, the following components are present:
```

Comment:
We now introduced the term "control node" for the machine that has these three components, so it might help to mention this here?